tfjs-backend-webgpu/src/kernels/matmul_packed_vec4_webgpu.ts

```diff
@@ -132,6 +132,7 @@ export class MatMulPackedVec4Program implements WebGPUProgram {
     this.dispatchLayout = {x: [2], y: [1], z: [0]};
     const vecSize = 4;
     if (outputShape[1] === 1) {
+      this.workGroupSize = [16, 1, 1];  // in case shared memory exceeds limit.
```

Comment:
Previously, when `outputShape[1] = 1`, the workGroupSize will be [64, 1, 1]. For the vec4 matmul, `mm_Bsub` will become `vec4 mm_Bsub[64*4][64]` which is  64K exceeding the limit 32K on windows.